app/models/export/ProductDraftWrapper.java

```diff
@@ -0,0 +1,266 @@
+package models.export;
```

Comment:
Important, in our company putting things into the sphere database is *import*. Rename this everywhere.